extensions/change-case/src/change-case.tsx

```diff
@@ -52,12 +52,15 @@ async function readContent(preferredSource: string) {
 }
 
 function modifyCasesWrapper(input: string, case_: CaseFunction) {
-  const modifiedArr: string[] = [];
+  const modifiedRawArr: string[] = [];
+  const modifiedMarkdownArr: string[] = [];
   const lines = input.split("\n");
   for (const line of lines) {
-    modifiedArr.push(case_(line));
+    const modified = line.length === 0 ? "\u200B" : case_(line);
```

Comment:
I think it's best to only have `modifiedMarkdownArr` contain the zws. The user probably shouldn't have to copy a zws.

Reply:
Ah, erics118, is my understanding correct that you add only a zws becasue CommonMark could be made to line feed with 2 spaces?

Reply:
I used a zws to create force a newline to be made. Just having two spaces doesn't work in Raycast.

I suggest just having:
```js
  for (const line of lines) {
    let modified = case_(line);
    modifiedRawArr.push(modified);
    modifiedMarkdownArr.push((modified.length === 0 ? "\u200B" : modified) + "\n");
  }
```

Reply:
i see. i fixed again plz check this again